charts/du/templates/statefulset.yaml

```
@@ -73,7 +73,7 @@ command:
       sleep 5
     done
 
-    yq --yaml-output ".f1u.socket = {bind_addr: '$POD_IP', ext_addr: '$external_address'}" "/etc/input-config/config.yml" >"/etc/output-config/config.yml"
+    yq eval -o yaml ".f1u.socket[0].bind_addr = \"$POD_IP\" | .f1u.socket[0].ext_addr = \"$external_address\"" "/etc/input-config/config.yml" >"/etc/output-config/config.yml"
```

Is there a reason to do the separate assignments and not just:

```suggestion
    yq eval -o yaml ".f1u.socket = [{\"bind_addr\": \"$POD_IP\", \"ext_addr\": \"$external_address\"}]" "/etc/input-config/config.yml" >"/etc/output-config/config.yml"
```

which is shorter and (IMO) clearer?